client/src/rr/dnssec/signer.rs

```diff
@@ -687,11 +782,114 @@ MC0CAQACBQC+L6pNAgMBAAECBQCYj0ZNAgMA9CsCAwDHZwICeEUCAnE/AgMA3u0=
         println!("pkey:\n{}", String::from_utf8(rsa_pem).unwrap());
 
         let key = KeyPair::from_rsa(rsa).unwrap();
-        let sig0key = key.to_sig0key_with_usage(Algorithm::RSASHA256,
-            KeyUsage::Zone).unwrap();
+        let sig0key = key.to_sig0key_with_usage(Algorithm::RSASHA256, KeyUsage::Zone)
+            .unwrap();
         let signer = Signer::sig0(sig0key, key, Name::root());
         let key_tag = signer.calculate_key_tag().unwrap();
 
         assert_eq!(key_tag, 28551);
     }
-}
+
+    // TODO: these tests technically came from TBS in trust_dns_proto
```

Comment:
sadly I needed to move this out of `proto` where TBS lives, it would be nice to move these back to live with it again.

Reply:
Note that when #200 is fixed, `TBS` should probably end up in the same crate as `Signer` (because we won't (and shouldn't) make a copy of the input to verify the signature of during verification).